configure.ac

```diff
@@ -116,7 +116,7 @@ AS_IF([test "x$esapi_sf" = "xyes"],
 
 AC_ARG_ENABLE(
   [fapi],
-  [AS_HELP_STRING([--with-fapi],
+  [AS_HELP_STRING([--enable-fapi],
```

Comment:
Changing this breaks those building packages for very little gain. While I prefer enable-fapi to with-fapi, I don't want to break downstream packagers for something this trivial.

Reply:
This only changes the help text from configure to match the actual parameter that's supported. `docs/BUILDING.md` already references `--enable-fapi`. I've got no objections to using `--with-fapi` but that's not what `AC_ARG_ENABLE` does, it should be `AC_ARG_WITH` instead which *would* change the behaviour for downstream packagers.

(I only noticed this when trying to explicitly disable FAPI in the Debian package and finding the help text was wrong.)

Reply:
This was actually right... See https://github.com/tpm2-software/tpm2-pkcs11/pull/824. I need to revert this one patch